binding/springamqp/src/main/java/io/tracee/binding/springamqp/TraceeMessagePropertiesConverter.java

```diff
@@ -0,0 +1,74 @@
+package io.tracee.binding.springamqp;
+
+import com.rabbitmq.client.AMQP;
+import com.rabbitmq.client.Envelope;
+import io.tracee.Tracee;
+import io.tracee.TraceeBackend;
+import io.tracee.TraceeConstants;
+import io.tracee.configuration.TraceeFilterConfiguration;
+import io.tracee.transport.HttpHeaderTransport;
+import org.springframework.amqp.core.MessageProperties;
+import org.springframework.amqp.rabbit.support.DefaultMessagePropertiesConverter;
+
+import java.util.Arrays;
+import java.util.Map;
+
+import static io.tracee.configuration.TraceeFilterConfiguration.Channel.IncomingResponse;
+import static io.tracee.configuration.TraceeFilterConfiguration.Channel.OutgoingRequest;
+
+public class TraceeMessagePropertiesConverter extends DefaultMessagePropertiesConverter {
+
+	private final TraceeBackend backend;
+	private final HttpHeaderTransport transportSerialization;
+	private final String profile;
+
+	public TraceeMessagePropertiesConverter() {
+		this(Tracee.getBackend(), new HttpHeaderTransport(Tracee.getBackend().getLoggerFactory()), null);
+	}
+
+	public TraceeMessagePropertiesConverter(String profile) {
+		this(Tracee.getBackend(), new HttpHeaderTransport(Tracee.getBackend().getLoggerFactory()), profile);
+	}
+
+	protected TraceeMessagePropertiesConverter(TraceeBackend backend, HttpHeaderTransport transportSerialization, String profile) {
+		this.backend = backend;
+		this.transportSerialization = transportSerialization;
+		this.profile = profile;
+	}
+
+	/**
+	 * Incoming messages
+	 */
+	@Override
+	public MessageProperties toMessageProperties(AMQP.BasicProperties source, Envelope envelope, String charset) {
+		final MessageProperties messageProperties = super.toMessageProperties(source, envelope, charset);
+
+		final String header = String.valueOf(messageProperties.getHeaders().get(TraceeConstants.HTTP_HEADER_NAME));
```

Comment:
I think you should use the constant JMS_HEADER_NAME instead of HTTP_HEADER_NAME. (But we should rename JMS_HEADER_NAME to ASYNC_MESSAGE_HEADER_NAME afterwards)